CHANGES.rst

```diff
@@ -7,6 +7,10 @@
 
 - Host documentation at https://zopeappapidoc.readthedocs.io/
 
+- Add argument to ``static-apidoc`` for loading a specific ZCML file. To use this feature, the ZCML file you specify needs to 
+  establish a working Zope 3 publication environment. The easiest way to do so is to include this line in the ZCML:
+  ``<include package='zope.app.apidoc' file='ftesting.zcml' condition='have static-apidoc' />``. 
+  See `PR #13 <https://github.com/zopefoundation/zope.app.apidoc/pull/13/>`_.
```

Comment:
I don't know how I feel about 'ftesting.zcml' being the official entry point for a non-testing feature.  Could we introduce a 'static.zcml' and have ftesting.zcml include it?

Reply:
I agree. I was planning a followup PR, but if @jkmartindale wants to tackle it here that'd be great.